README.md

```diff
@@ -11,3 +11,32 @@ page](https://docs.github.com/en/get-started/writing-on-github/working-with-adva
 might help with the notation for mathematical expressions.
 
 $T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0$
+
+## Proof that $O(\log_2 n) = O(\log_5 n)$
+
+
+1. **Big-O Definition**: $f(n) \in O(g(n)) \iff \exists c > 0, n_0 \geq 1: f(n) \leq c \cdot g(n) \ \forall n \geq n_0$.
+
+2. **Change of Base Formula**: $\log_a n = \frac{\log_b n}{\log_b a}$.
+
+
+
+## Proof:
+
+To prove $O(\log_2 n) = O(\log_5 n)$, have to show that $\log_2 n \in O(\log_5 n)$ and $\log_5 n \in O(\log_2 n)$. First, considering $\log_2 n \in O(\log_5 n)$. Using the change of base formula, $\log_2 n = \frac{\log_5 n}{\log_5 2}$. Let $c = \frac{1}{\log_5 2} > 0$ (since $\log_5 2$ is a positive constant). Then $\log_2 n \leq c \cdot \log_5 n \ \forall n \geq 1$, which satisfies the definition of $\log_2 n \in O(\log_5 n)$. 
```

Comment:
You cannot assume any particular $T(n)$.

Reply:
I do not beleive I assumed any I just proved the case that was presented? So youre asking for a broad and do not mention the example that was given?